pkg/traces/config.go

```diff
@@ -901,6 +902,7 @@ func tracingFactories() (otelcol.Factories, error) {
 	}
 
 	receivers, err := receiver.MakeFactoryMap(
+		awsfirehosereceiver.NewFactory(),
```

Comment:
I think we don't need this line; it would only make sense for static mode where we haven't wired up the rest of the code or documented as an option. cc @ptodev